README.rst

```diff
@@ -95,6 +242,8 @@ Updating a Replication Set
 Dropping a Replication Set
 --------------------------
 
+Replication sets may be dropped via the ``drop`` command.
```

Comment:
I didn't actually know when I wrote this.  @tkaemming may have more info, but looking at it now the `administration.drop_set` code it starts a ZK transaction, then using a postgres prepared transaction calls `administration.unconfigure_set`, which removes the trigger and PgQ from the database.

As far as the relay is concerned I think it probably would error?  As the queue it was attempting to relay from just disappeared?  The [docs for the `pgq.drop_queue` operation](http://skytools.projects.pgfoundry.org/skytools-3.0/pgq/files/external-sql.html#pgq.drop_queue(1)) do say, "No consumers must be listening on the queue," so that seems to back up my guess.